From 4bb30cefbdef1b05729dd5ceae418e7548a98092 Mon Sep 17 00:00:00 2001
From: Joseph Chen <chenjh@rock-chips.com>
Date: Thu, 6 Sep 2018 10:44:00 +0800
Subject: [PATCH] arm: dts: rk3308-dot-rk816-v10-aarch32: update battery
 information

The board battery is attached, so update battery hardware info.

Change-Id: Ife9630c61c712d21ff4461cb53175833b3d4c779
Signed-off-by: Joseph Chen <chenjh@rock-chips.com>
---
 arch/arm/boot/dts/rk3308-dot-rk816-v10-aarch32.dts | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/rk3308-dot-rk816-v10-aarch32.dts b/arch/arm/boot/dts/rk3308-dot-rk816-v10-aarch32.dts
index e3a3447d7b99..7f9ea8f04109 100644
--- a/arch/arm/boot/dts/rk3308-dot-rk816-v10-aarch32.dts
+++ b/arch/arm/boot/dts/rk3308-dot-rk816-v10-aarch32.dts
@@ -212,11 +212,11 @@
 			ocv_table = <3400 3625 3685 3697 3718 3735 3748
 				     3760 3774 3788 3802 3816 3834 3853
 				     3877 3908 3946 3975 4018 4071 4106>;
-			design_capacity = <5000>;
-			design_qmax = <5200>;
-			bat_res = <100>;
+			design_capacity = <2750>;
+			design_qmax = <3000>;
+			bat_res = <110>;
 			max_input_current = <2000>;
-			max_chrg_current = <1300>;
+			max_chrg_current = <1800>;
 			max_chrg_voltage = <4200>;
 			sleep_enter_current = <300>;
 			sleep_exit_current = <300>;
@@ -225,7 +225,7 @@
 			zero_algorithm_vol = <3850>;
 			max_soc_offset = <80>;
 			monitor_sec = <5>;
-			virtual_power = <1>;
+			virtual_power = <0>;
 			power_dc2otg = <0>;
 			dc_det_adc = <0>;
 		};
-- 
2.35.3

